src/__tests__/LDClient-events-test.js

```diff
@@ -274,6 +274,7 @@ describe('LDClient events', () => {
         await client.identify(user1);
 
         expect(ep.events.length).toEqual(0);
+        expect(platform.testing.logger.output.warn).toEqual([messages.doNotTrackEnabled()])
```

Comment:
There's a failing GitHub check because this is missing a trailing semi-colon.

Reply:
Actually, rather than adding an assertion to this existing test case, it would be preferable to have a new test case. That way we can trigger multiple events and additionally verify that the Do-Not-Track message is only logged once.

Reply:
> That way we can trigger multiple events and additionally verify that the Do-Not-Track message is only logged once.

Makes sense.

> it would be preferable to have a new test case.

Hmm. Thing is, this test case would basically be a copy-and-paste of the existing test case but with extra lines. If one test is going to basically be a subset of the other, it probably makes sense to just have the larger one?

With that in mind I've extended the existing test case to make a second `.identify` call and then check that the logger output is unchanged.

Possibly we've got slightly different philosophies about test suite design and this won't seem like the right solution to you; if so, feel free to tweak as you see fit! *"Allow edits by maintainers"* is switched on. :)

Reply:
Yeah, this very well could be due to slight differences in test philosophies. The perspective I'm coming from is that, in the case that the test fails, I want the test name to denote what went wrong. That wouldn't be the case here -- we'd be testing the behavior of the logging, rather than that the SDK "does not send an identify event if doNotTrack is set". Also the behavior we're testing isn't specific to identify events -- for example we also have a test in this file to verify that track events aren't sent if Do-Not-Track is enabled.

As for the copy-paste concern, the test case I'm thinking of is _slightly_ different -- effectively we'd have two calls to generate events before making any assertions.. For example, two `client.identify` calls, or maybe one `client.identify` and one `client.track`, followed by just a single assertion for the single logged message.